test/evp_test.c

```diff
@@ -2504,8 +2504,8 @@ static int keygen_test_run(EVP_TEST *t)
 {
     KEYGEN_TEST_DATA *keygen = t->data;
     EVP_PKEY *pkey = NULL;
+    int rv = 1;
 
-    t->err = NULL;
     if (EVP_PKEY_keygen(keygen->genctx, &pkey) <= 0) {
         t->err = "KEYGEN_GENERATE_ERROR";
         goto err;
```

Comment:
not sure I see the point of all this setting rv ... May as well just return 1 instead of goto err;

Reply:
Consistent code.  This is how it's done in the rest of the file, it follows a common pattern that makes it easy to follow.

Reply:
hooray!